packages/api-reference/src/hooks/useSpec.test.ts

```diff
@@ -101,6 +101,7 @@ describe('useSpec', () => {
 
     await new Promise((resolve) => {
       watch(rawSpecRef, (value) => {
+        if (!value) return
```

Comment:
@hanspagel there was no value in here which was causing an error. Is this value allowed to be falsey?